src/configuration/services/mongodb.md

```diff
@@ -115,3 +118,7 @@ mongodump --port 30000 -u main -p main --authenticationDatabase main --db main
 As with any other shell command it can be piped to another command to compress the output or redirect it to a specific file.
 
 For further references please see the [official mongodump documentation](https://docs.mongodb.com/manual/reference/program/mongodump/#bin.mongodump).
+
+## Upgrading
```

Comment:
I would also mention that upgrading only consists of updating the image version tag, and no further steps should be necessary, provided (this is important) the application does not make use of any version specific features (ie, when upgrading to 3.2, the app is not using any 3.0 specific features that are not compatible with 3.2, and so on).